frontend/src/member/MemberPage.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import {MemberModel} from "./MemberModel";
+import styled from "styled-components";
+import MemberBoard from "./MemberBoard";
+import {NavLink} from "react-router-dom";
+
+const baseUrl = '/api/members/';
+
+export default function MemberPage() {
+    const [memberList, setMemberList] = useState<MemberModel[]>([])
+
+    useEffect(() => {
+        fetchAllMembers()
+    }, [])
+
+    const fetchAllMembers = () => {
+        axios.get(baseUrl)
+            .then((response) => response.data)
+            .catch((error) => {
+                console.log('[Error von GET]: =>' + error)
+            })
+            .then((data) => {
+                setMemberList(data)
+            })
+    }
+
+    const memberListOnBoard = memberList.map(member => {
+        return <MemberBoard key={member.id} member={member} fetchAllMembers={fetchAllMembers}/>
+    })
+
+
+     return <>
+         <StyledButton>
+             <NavLink to="/">Go to MainPage</NavLink></StyledButton>
+         <StyledButton>
+             <NavLink to="/add/members">Add MemberPage</NavLink></StyledButton>
```

Comment:
Bitte die Formulierungen überarbeiten. "Page" ist vielleicht nicht allen Anwender*innen geläufig.

```suggestion
             <NavLink to="/add/members">Add member</NavLink></StyledButton>
```